actors/power/src/state.rs

```diff
@@ -103,26 +103,26 @@ impl State {
         &self,
         policy: &Policy,
         s: &BS,
-        miner: &Address,
-    ) -> anyhow::Result<bool> {
+        miner: ActorID,
+    ) -> anyhow::Result<(StoragePower, bool)> {
```

Comment:
Sorry, I was vague earlier. Change this to return `Result<(..), ActorError>`. You'll then be able to specify different exit codes at the `?` operators below (and just `?` up at the call site).